DESCRIPTION

```diff
@@ -17,6 +17,8 @@ Imports:
     dplyr (>= 0.2),
     stringi,
     lazyeval
+Suggets:
```

Comment:
I meant to add data.table to the suggested packages to address this [build failure](https://travis-ci.org/hadley/tidyr/builds/48924616), but listing it under a redundant/misspelled key was just a silly mistake (insert embarrassed emoji here).